webpack/assets/javascripts/react_app/components/LoadingState/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { LoadingState as PfLoadingState } from 'patternfly-react';
```

Comment:
I don't like this component, its throwing errors for me, will add a default `LoadingComponent` in #6767